src/GeneralStateTestsFiller/stEWASMTests/callValueExceeds64bitFiller.yml

```diff
@@ -0,0 +1,94 @@
+callValueExceeds64bit:
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: '0x020000'
+    currentGasLimit: '89128960'
+    currentNumber: '1'
+    currentTimestamp: '1000'
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+  pre:
+    deadbeef00000000000000000000000000000000:
+      balance: '0xffff'
+      nonce: ''
+      storage: {}
+      code: |
+        (module
+          (import "ethereum" "call" (func $call (param i64 i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "getBalance"  (func $getBalance (param i32 i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (import "ethereum" "getAddress" (func $getAddress (param i32)))
+          (memory 1 )
+          (data (i32.const 0)  "\01\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\ef\be\ad\de")
+          (data (i32.const 20) "\00\00\00\00\00\00\00\00\00\ff\ff\ff\ff\ff\ff\ff\ff\ff\ff\ff")
+          (data (i32.const 52) "\42\04\20\42\04\20\42\00")
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+          (func $main
+            ;; Memory layout:
+            ;;   0 -  20 bytes: address (4)
+            ;;  20 -  52 bytes: value (0)
+            ;;  52 -  56 bytes: data (0x42004200)
+            ;;  56 -  60 bytes: result
+            (i32.store (i32.const 56) ;; store the result so it is not returned
```

Comment:
Here you are storing the result into the wasm memory, but you  still need to store it in the contract storage, to do that you have to use the storageStore eei method, which receives key offset, and value offset, 

for example: if you want to store in the key 0 of the storage you have to send an offset which have the value 0, in this case you could use offset 60, and I think you want to store the result which is in offset 56. so you could call storageStore method like this:

```
(call $storageStore (i32.const 60) (i32.const 56))
```